### ISSUE TITLE
MissingArgs raised when TypeError raised during command invocation

### ISSUE THREAD

**Issue by neonjonn** [Jonxslays/Yami#33]
**Describe the bug**
`yami.exceptions.MissingArgs` was raised when a `TypeError` was raised during a command invocation

![error](https://media.discordapp.net/attachments/765236395689902111/928704009362092032/unknown.png)

**To Reproduce**
Raise `TypeError` in a command (although I had an actual mistake in this case)

**Expected behavior**
Not really sure 😅 

**System Information:**
```
Yami v0.3.0.post0 [HEAD]
@ some_stuff\.venv\lib\site-packages\yami
CPython 3.10.1 MSC v.1929 64 bit (AMD64)
Windows DESKTOP-6HI8J39 10 AMD64
10.0.22000
```

**Additional context**
None

**Comment by Jonxslays** [comment]
After some thought based on the fact that your command **is actually** missing a required arg, this exception seems valid.

**Comment by Jonxslays** [comment]
This actually is a big seeing as how the argument that was missed was not a parameter of the command, but of a function inside the command. Thanks for the report.

**Comment by Jonxslays** [comment]
This is fixed in dce84ee.

Not yet released, but feel free to install from git with:
`pip install git+https://github.com/Jonxslays/Yami.git@dce84eeb51e98b88ba2ba6ea92f81f4eebe5c785`

I will close this issue and let you know when it's on PyPI.

**Comment by Jonxslays** [comment]
Resolved, and released in [v0.3.1](https://github.com/Jonxslays/Yami/releases/tag/v0.3.1). :)
